xml/snmp_sysdescr.xml

```diff
@@ -2322,6 +2322,21 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="1" name="os.version"/>
   </fingerprint>
 
+  <!--======================================================================
+                            Fortinet FortiGate
+   =======================================================================-->
+
+  <fingerprint pattern="^[\w\-]+\s+v([\d\.]+),build\d+,\d+(?:\s+\(\w+\))?$">
+    <description>FortiGate</description>
+    <example os.version="6.2.2">FortiaGate-61F v6.2.2,build6083,19102</example>
```

Comment:
Is "FortiaGate" the value that appears in the SNMP data or is that a typo for "FortiGate"?